lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1311,7 +1315,7 @@ class ConvertElementwiseOp : public ConversionPattern {
   LogicalResult
   matchAndRewrite(Operation *op, ArrayRef<Value> operands,
                   ConversionPatternRewriter &rewriter) const override {
-    if (!isa<AtenTanhOp, AtenReluOp, AtenPreluOp, AtenGeluOp,
+    if (!isa<AtenTanOp, AtenTanhOp, AtenReluOp, AtenPreluOp, AtenGeluOp,
```

Comment:
You also need to add the op to

https://github.com/llvm/torch-mlir/blob/6fbd8ea0c8a97e57d03cd3598cef5941d14f93c4/lib/Conversion/TorchToLinalg/Uncategorized.cpp#L1966

Reply:
I believe I have the remaining boilerplate based on the sample PR.